client/src/pages/Home/Forms/DonatorFormReqest/DonatorForm.js

```diff
@@ -1,13 +1,93 @@
 import React from "react";
+import { useState } from "react";
 import "./DonatorForm.css";
 
 function DonatorForm() {
+	const [name, setName] = useState("");
+	const [address, setAddress] = useState("");
+	const [numberOfLaptops, setNumberOfLaptops] = useState("");
+	const [phoneNumber, setPhoneNumber] = useState("");
+	const [email, setEmail] = useState("");
+	const [deliveryOption, setDeliveryOption] = useState("");
+
+
+	function handleClick(e) {
+		if (e.target.name === "name") {
+			setName(e.target.value);
+		} else if (e.target.name === "address") {
+			setAddress(e.target.value);
+		} else if (e.target.name === "email") {
+			setEmail(e.target.value);
+		} else if (e.target.name === "phoneNumber") {
+			setPhoneNumber(e.target.value);
+		} else if (e.target.name === "numberOfLaptops") {
+			setNumberOfLaptops(e.target.value);
+		}else if (e.target.name === "deliveryOption") {
```

Comment:
ask vs code to run the auto formatting (I think it's right click -> format this document)

Reply:
I have used prettier to format the code.